literature.bib

```diff
@@ -4199,6 +4445,16 @@ @article{RTS+:JLAMP16
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2015 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{SH15,
+  title={{Variability Exchange Language-A Generic Exchange Format for Variability Data}},
+  author={Schulze, Michael and Hellebrand, Robert},
+  booktitle={Software Engineering (Workshops)},
+  pages={71--80},
+  year= 2015
+}
```

Comment:
Is this a `@book`?

Reply:
No, but I cannot find the conference whatsoever.

Reply:
Ok, 🤷‍♂️ .